app/models/concerns/hyrax/collection_behavior.rb

```diff
@@ -111,18 +111,15 @@ def collection_type_gid_document_field_name
       end
     end
 
-    # Compute the sum of each file in the collection using Solr to
-    # avoid having to access Fedora
-    #
-    # @return [Fixnum] size of collection in bytes
-    # @raise [RuntimeError] unsaved record does not exist in solr
+    # @deprecated to be removed in 4.0.0; this feature was replaced with a
+    #   hard-coded null implementation
+    # @return [Fixnum] 0
     def bytes
-      return 0 if member_object_ids.empty?
-
-      raise "Collection must be saved to query for bytes" if new_record?
-
-      # One query per member_id because Solr is not a relational database
-      member_object_ids.collect { |work_id| size_for_work(work_id) }.sum
+      Deprecation.warn("#bytes has been deprecated for removal in Hyrax 4.0.0; " \
```

Comment:
I'm a little surprised Rubocop didn't flag this for using double-quotes. I don't care deeply about this, but thought I'd raise it to see if you do.

Reply:
Bixby don't care about how many quotes you use.